docs/components/Button.md

```diff
@@ -8,7 +8,27 @@ Button is a component used to interact with an application.
 ## Example
 
 ```javascript
-<Button />
+class MyApp extends React.Component {
+
+  buttonClickHandler = (event) => {
+     console.log("Hello from buttonClickHandler");
+  }
+
+  render () {
+    return (
+      <View>
```

Comment:
do you need `View`?

Reply:
In this snippet NO since there is only node but in any other more complex scene graph YES.

Reply:
It is not required, but most of the time you want to render more then one component. That's why I left <View>